nistats/hemodynamic_models.py

```diff
@@ -343,16 +343,16 @@ def compute_regressor(exp_condition, hrf_model, frametimes, con_id='cond',
     'canonical_derivative': the Glover hrf + time derivative (2 regressors)
     'fir': finite impulse response basis, a set of delayed dirac models
            with arbitrary length. This one currently assumes regularly spaced
-           frametimes (i.e. fixed time of repetition).
+           frame_times (i.e. fixed time of repetition).
```

Comment:
Same kind of comments as previous docstrings: do we want to use some words here or the variable name.